style/git_and_github.qmd

```diff
@@ -1,17 +1,88 @@
-# Using git and GitHub
+# Using Git and GitHub
 
-* Commits should be [atomic](https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84)
-* All code should be committed via pull request to main. **Never** push to main
+-   Commits should be [atomic](https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84)
+-   All code should be committed via pull request (PR) to `main`
```

Comment:
All code committed to `main` should be via pull request (PR)

Thoughts on above word order as you can commit to a branch without needing a PR. Would this make it more clear or less clear?

Reply:
I think the whole sentence is somewhat clumsy actually. It's somewhat contradicting what are saying in the callout of never push to main. Yes, strictly a PR is just a merge, which is a commit into main, but it's not a commit you are making directly.